mypy/plugin.py

```diff
@@ -170,12 +170,12 @@ def fail(self, msg: str, ctx: Context, *, code: ErrorCode | None = None) -> None
         raise NotImplementedError
 
     @abstractmethod
-    def named_type(self, name: str, args: list[Type]) -> Instance:
+    def named_type(self, fullname: str, args: list[Type], /) -> Instance:
         """Construct an instance of a builtin type with given name."""
         raise NotImplementedError
 
     @abstractmethod
-    def analyze_type(self, typ: Type) -> Type:
+    def analyze_type(self, typ: Type, /) -> Type:
```

Comment:
Same case here. The `/` feels unnecessary. Changing the argument name to `typ` in the single implementation would probably be better.
https://github.com/python/mypy/blob/ac6151ae6657a8bbd9793c74691c9985dae02c15/mypy/typeanal.py#L1765-L1766